test/users/engin/compiler/innerRecord.future

```diff
@@ -0,0 +1,8 @@
+bug: nil dereference referencing outer class' fields
+
+NOTES
+
+- Where outer,inner == class,class; same thing happens, which I would understand
+  due to memory allocation differences btw records vs classes
```

Comment:
Hmm, is the nil dereference on the same line in the class/class case?  I would expect to receive a nil dereference on outerClass.r (or whatever the name of the stored inner class is) rather than within the inner class's method, since the default value of a class is nil.